particles/Tutorial/Javascript/Demo/TTTGame.arcs

```diff
@@ -4,133 +4,134 @@ import 'TTTGameSchemas.arcs'
 import 'TTTGameData.arcs'
 
 particle Game in './source/TTTGame.js'
-  inout Person {name, avatar, id} playerOne
-  inout Person {name, avatar, id} playerTwo
-  inout GameState gameState
-  inout Move {move} playerOneMove
-  inout Move {move} playerTwoMove
-  inout [Event] events
-  consume root
-    provide gameSlot
+  playerOne: reads writes Person
+  playerTwo: reads writes Person
+  gameState: reads writes GameState
+  playerOneMove: reads writes Move
+  playerTwoMove: reads writes Move
+  events: reads writes [Event]
+  root: consumes
+    gameSlot: provides
 
 particle TicTacToeBoard in './source/TTTBoard.js'
-  in GameState {board, currentPlayer} gameState
-  out [Event] events
-  consume boardSlot
+  gameState: reads GameState
+  events: writes [Event]
+  boardSlot: consumes
 
 particle HumanPlayer in './source/TTTHumanPlayer.js'
-  in GameState {currentPlayer} gameState
-  in [Event] events
-  in Person {id} player
-  out Move {move} myMove
+  gameState: reads GameState
+  events: reads [Event]
+  player: reads Person
+  myMove: writes Move
 
 particle RandomComputer in './source/TTTRandomComputer.js'
-  out Move {move} computerMove
-  in GameState {board, currentPlayer} gameState
-  in Person {id} player
+  computerMove: writes Move
+  gameState: reads GameState
+  player: reads Person
 
 recipe GameDemoRecipe
-  create #volatile #humanMove as playerOneMove
-  create #volatile #computerMove as playerTwoMove
-  create #volatile as gameState  
-  use ComputerStore as playerTwo
-  use HumanStore as playerOne
+  playerOneMove: create
+  playerTwoMove: create
+  gameState: create
+  playerTwo: use ComputerStore
+  playerOne: use HumanStore
 
   Game
-    consume root 
-      provide gameSlot as board
-    gameState = gameState
-    playerOne = playerOne
-    playerTwo = playerTwo
-    playerOneMove = playerOneMove
-    playerTwoMove = playerTwoMove
-    events = events
+    root: consumes
+      gameSlot: provides board
+    gameState: any gameState
```

Comment:
@Cypher1 can you try to chase down whether we could get rid of any or use no capability as default. e.g. `gameState: gameState`.

Reply:
Talking to Shane about it tomorrow, having checked the parser, this seems like a clear win and low hanging fruit for readability improvements.

Reply:
in addition to this, could we use a conventions naming handles/slots `fooHandle`/`barSlot`? 
otherwise this list of identical names looks weird:
`foo: foo
  bar: bar
...`
ideally we would omit them altogether and have the recipe resolver fill them in, but probably now isn't the right time for this.